src/client/handlers/vaultsCreate.ts

```diff
@@ -28,4 +28,4 @@ class VaultsCreatehandler extends UnaryHandler<
   }
 }
 
-export { VaultsCreatehandler };
+export { VaultsCreateHandler };
```

Comment:
I intentionally avoided the default export here to enforce the name of the class when importing. Keep in mind, with default exports you can import them with any name, E.G. `import SomeRandomName from './VaultsCreate'` is valid. Since these handlers are imported in large blocks I wanted to avoid a situation where the import names could detach from the actual class name if we're not being careful.

Reply:
It was just convention when there's single class files. It's only lower case names when you intend to export other things. We use this standard elsewhere.

Reply:
We can deal with this later. It was because of legacy from GRPC.